modules/oxide/src/less/theme/content/resize-handles/resize-handle.less

```
@@ -61,7 +61,7 @@
 
   .mce-resize-helper {
     background: #555;
-    background: rgba(0, 0, 0, .75);
+    background: rgba(0 0 0 75%);
```

Doesn't this need a `/`?
```suggestion
    background: rgba(0 0 0 / 75%);
```